taskcluster/ci/config.yml

```diff
@@ -141,6 +141,12 @@ taskgraph:
             default-repository: git@github.com:mozilla-extensions/firefox-quick-suggest-weather
             default-ref: master
             type: git
+        quicksuggestcalculator:
+            name: "Firefox Quick Suggest Calculator"
+            project-regex:  firefox-quick-suggest-calculator$
+            default-repository: git@github.com:mozilla-extensions/firefox-quick-suggest-calculator
```

Comment:
This isn't a private repo, so this needs to be an https url, per https://github.com/mozilla-extensions/xpi-manifest/blob/master/docs/adding-a-new-xpi.md#enabling-releases